src/benchmarks/micro/corefx/System.Buffers/ArrayPoolTests.cs

```diff
@@ -13,6 +13,7 @@ namespace System.Buffers.Tests
     [BenchmarkCategory(Categories.CoreFX)]
     [GenericTypeArguments(typeof(byte))] // value type
     [GenericTypeArguments(typeof(object))] // reference type
+    [GcServer(true)] // Server GC is the most important scenario for ArrayPool
```

Comment:
Did you confirm this made it switch to server GC? 
Somehow a similar switch did not work for me, but it is possible I did something else wrong. Just want to confirm this works when used properly.